ferveo/src/lib.rs

```diff
@@ -419,28 +430,37 @@ mod test_dkg_full {
                     .decryption_key;
 
                 // Creates updated private key shares
+                // TODO: Why not using dkg.aggregate()?
```

Comment:
IIRC `dkg.aggregate` would throw if we didn't feed enough transcripts into it. `dkg` has an internal state machine that tracks the progress of the DKG ritual, and we would have to strictly adhere to its protocol in order to successfully `aggregate` here. Since refreshing/recovery is not implemented in the said state machine, it's easier to skip different DKG states and just call `aggregate` directly here.